s3/main.py

```diff
@@ -9,10 +9,10 @@
 app = Flask(__name__)
 
 
-@app.route("/get/<service_id>/", methods=["GET"])
-def get_file(service_id):
+@app.route("/<service_id>", methods=["GET", "DELETE"])
+def s3_file(service_id):
```

Comment:
I think it makes sense to separate the two methods into separate functions

Reply:
Why is that? Right now both preform a simple get/delete. Possibly for future changes?

Reply:
Those are different operation and the framework provides a way to separate them cleanly rather than have to explicitly branch in your code.

Reply:
Request methods split.